internal/inventory/tagprocessor.go

```diff
@@ -94,6 +94,7 @@ func (tp *TagProcessor) ProcessReport(r *llrp.ROAccessReport, info ReportInfo) (
 	}
 
 	for _, rt := range r.TagReportData {
+		//nolint: gosec
 		if event := tp.processData(&rt, info); event != nil {
```

Comment:
I think this one needs to be fix  to use `&r.TagReportData[index]`

Reply:
Yep, I just made the change